src/core/utils.js

```diff
@@ -582,3 +582,27 @@ export function addId(elem, pfx = "", txt = "", noLC = false) {
   elem.id = id;
   return id;
 }
+
+/**
+ * Returns all the descendant text nodes of an element.
+ * @param {Node} elem
+ * @param {Array:String} exclusions node types (localName) to exclude
+ * @returns {Array:String}
+ */
+export function allTextNodes(elem, exclusions) {
+  const excl = new Set(exclusions);
```

Comment:
should move this outside, me thinks... no point in creating it over and over again.

Reply:
make user call with set instead of array?